ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -353,24 +394,28 @@ export const AdminInstitutionEdit = withUrlParams()(class extends React.Componen
             <label style={styles.label}>Agreement Type</label>
             <Dropdown style={{width: '16rem'}} data-test-id='agreement-dropdown'
                       placeholder='Your Agreement'
-                      options={DuaTypes}
-                      value={institution.duaTypeEnum}
-                      onChange={v => this.setState(fp.set(['institution', 'duaTypeEnum'], v.value))}/>
-            {institution.duaTypeEnum === DuaType.RESTRICTED && <FlexColumn data-test-id='emailAddress' style={{width: '16rem'}}>
+                      options={MembershipRequirements}
+                      value={getRegisteredTierRequirement(institution).membershipRequirement}
+                      onChange={(v) => this.setTierRequirement(v, 'tierRequirements')}/>
+            {getRegisteredTierRequirement(institution).membershipRequirement === InstitutionMembershipRequirement.ADDRESSES &&
```

Comment:
add TODO CT comment

Reply:
how to add comment here. I tried htlm comment and js comment, both get compile error....

Reply:
Still slightly feel the TODO is not necessary because it should be everywhere in this class if we decide to add...

Reply:
For the record:

```
{/* my comment */}
```

If there's a lot of instance though, agree just a TODO at the top of the component may suffice here